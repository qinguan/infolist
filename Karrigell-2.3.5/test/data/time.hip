import time
today = time.strftime("%d:%m:%y",time.localtime(time.time()))
"The current date is\n"+today
