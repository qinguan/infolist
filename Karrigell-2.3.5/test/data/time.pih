The current date is
<% import time
print time.strftime("%d:%m:%y",time.localtime(time.time()))
%>
