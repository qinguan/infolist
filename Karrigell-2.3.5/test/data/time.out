The current date is
%(date)s
